Proton/Sources/Swift/Core/RichTextView.swift

```diff
@@ -117,6 +117,26 @@ class RichTextView: AutogrowingTextView {
         getNestedEditors(for: self)
     }
 
+    override public func pressesBegan(_ presses: Set<UIPress>, with event: UIPressesEvent?) {
+        guard #available(iOS 13.4, *) else { return }
+        var handled: Bool = false
+        presses.forEach { press in
+            guard let key = press.key else { return }
+            if key.charactersIgnoringModifiers == UIKeyCommand.inputUpArrow {
```

Comment:
@hontd, if we keep a mapping of `UIKeyCommand` with `EditorKey`, we can get this if-else condition down to a single line without duplicating ` richTextViewDelegate?.richTextView(self, shouldHandle:`. It would be easier to maintain then.

Reply:
Also, it would be great to add some tests here. Checkout `RichTextViewContextTests` for examples.